### ISSUE TITLE
Realtek RTL8152/RTL8153/RTL8156 USB 2.5G Adapters not working

### ISSUE THREAD

**Issue by wangzhuok** [fbelavenuto/arpl#458]
Please add/fix it , THX !!!

**Comment by crazyshitass** [comment]
Same here, I tried many methods——select/unselect driver modules 、change lan port number in DSM7、rebuild loader…all in vain. 
Here are same similar cases:
https://github.com/fbelavenuto/arpl-modules/issues/72
https://wp.gxnas.com/12245.html/comment-page-10
https://github.com/fbelavenuto/arpl-modules/issues/67
https://github.com/fbelavenuto/arpl-modules/issues/11
https://www.mi-d.cn/2396

**Comment by stonebreaker2019** [comment]
how about Intel 225/226 adapter?

**Comment by crazyshitass** [comment]
> 

onboard NIC may works fine

**Comment by wangzhuok** [comment]
i test RTL8125B PCIE 2.5G adapter can running , RTL8153 USB 1G adapter can running , but RTL8156 USB 2.5g adapter is not working, i update arpl and try unselect all rtl drivers , not working , then install spk driver , not working .

**Comment by crazyshitass** [comment]
> i test RTL8125B PCIE 2.5G adapter can running , RTL8153 USB 1G adapter can running , but RTL8156 USB 2.5g adapter is not working, i update arpl and try unselect all rtl drivers , not working , then install spk driver , not working .

Yes, loader's kernel or  rtl8156b driver should cause the problem.
